src/games/rcll/challenges.clp

```diff
@@ -7,82 +7,466 @@
 ;  Licensed under BSD license, cf. LICENSE file
 ;---------------------------------------------------------------------------
 
+(defglobal
+	?*BASE_STATION* = 1
```

Comment:
CLIPS version of an enum?

Reply:
exactly :P to mirror the enum for the mps generation^^